subgraph/abis/PublicLockV11.json

```diff
@@ -0,0 +1,2314 @@
+{
```

Comment:
I am not a huge fan of adding these files again since they are already in the `contracts` package of the monorepo. I worry they would be out of sync.

Reply:
Have removed and went back to using the `prepare:abis` script which works with current implementation.